dali/server/daserver.cpp

```diff
@@ -418,128 +418,131 @@ int main(int argc, char* argv[])
                     recursiveCreateDirectory(dataPath.str());
             }
 
-            // JCSMORE remoteBackupLocation should not be a property of SDS section really.
-            StringBuffer mirrorPath;
-            if (!getConfigurationDirectory(serverConfig->queryPropTree("Directories"),"mirror","dali",serverConfig->queryProp("@name"),mirrorPath)) 
-                serverConfig->getProp("SDS/@remoteBackupLocation",mirrorPath);
-
-            if (mirrorPath.length())
+            if (serverConfig->getPropBool("SDS/@backupEnabled", true))
```

Comment:
NB: this is only real change in this file, rest is due to prev. code being in indented block.